lib/3scale_toolbox/commands/import_command/openapi/threescale_api_spec.rb

```diff
@@ -44,6 +45,10 @@ def security
             @security ||= parse_security
           end
 
+          def prefix_matching
```

Comment:
this is exactly what ` attr_reader :prefix_matching`  does

Reply:
:+1: